src/products/mozilla_org/manage_mozillaorg.md

```diff
@@ -0,0 +1,95 @@
+# Managing Mozilla.org
+
+Since the migration from .lang to Fluent, the L10n PM’s job is a lot simpler. The PM’s primary focus is on reviewing the string extractions provided by the bedrock team, provides feedback, manages the project through Pontoon and communicates important information to the communities.
+
+## Reviewing the PR
+
+When a PR is ready for localization review, through a script, the PR is pushed to the [www-l10n](https://github.com/mozilla-l10n/www-l10n/tree/master/) repository. The naming convention of the string ID is the file name followed by the first few words of the string, up to 7 words.
+
+### Reviewing new file
+
+* `en` is the source of a localizable file; en-US is a locale. Review is conducted in `en`.
+* Check that the demo URL works so you can review the text against the page on staging.
+* Check that all the brand and product names are in the form of placeable. If not, make sure that the term exists or it needs to be added.
+* Check for expressions that are metaphors or play with words. Be sure to include comments for alternatives.
+* Check for consistent use of punctuation in a series of bullet items or parallel items.
+* Check for proper use of punctuation, especially long sentences.
+* Check variables and references to the variables in the string.
+
+### Reviewing updated file
+
+* A revised string usually has a unique string ID suffix: `update`. The news string is right before the string replaced.
+* When a new string is added, such as `firefox-new-page-title-updated`, it will only be added to the `en` file in the PR, as only locales that have translated the string should contain it.
```

Comment:
> as only locales that have translated the string should contain it.

Is that the case? I assume we always add the strings only to `en`, this is not a special case.